libsupport/include/katana/CpuGpuSwitch.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#ifdef KATANA_ENABLE_GPU
+#define KATANA_GPU_ENABLED true
+#define KATANA_MEMORY_PIN_TYPE (katana::MemoryPinType::Pinned)
+#else
+#define KATANA_GPU_ENABLED false
+#define KATANA_MEMORY_PIN_TYPE (katana::MemoryPinType::Usual)
+#endif  // KATANA_ENABLE_GPU
```

Comment:
This isn't used yet. I would rather add this file in a later PR that actually uses it.

Reply:
This is currently used in https://github.com/KatanaGraph/katana-enterprise/pull/1551 . I had to place it in `katana` rather than `katana-enterprise` repo because someone may use the constants for `DynamicBitset` and `PODVector` instances in `katana` repo too.

Reply:
I don't really understand this argument. These values only have meaning for GPU enabled builds, and they should really be dynamic parameters or allocators anyway not build parameters, so hopefully we will be removing them in a refactor soon. So I see no reason to allow people to use them in more places that we will have to fix later.